custom_components/frigate/media_source.py

```diff
@@ -1273,38 +1244,17 @@ def _count_by(
 
     @classmethod
     def _generate_recording_title(
-        cls, identifier: RecordingIdentifier, folder: dict[str, Any] | None = None
+        cls, identifier: RecordingIdentifier, next_part: str | None = None
```

Comment:
I'm suspecting it might be cleaner to get rid of this method entirely, and just have the relevant strptime call in the method that is generating the media items?